src/defaultConfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "options": {
+    "separator": "@"
```

Comment:
i'm liking `--` as a default, faster to type than `@`

Reply:
`--` is used as a flag prefix though, might be better to choose something else